src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -11,9 +11,20 @@ function min_Δxyz(grid)
     Δy = minimum_grid_spacing(grid.Δy, topo[2])
     Δz = minimum_grid_spacing(grid.Δz, topo[3])
 
-    return min(Δx, Δy, Δy)
+    return min(Δx, Δy, Δz)
```

Comment:
PS.: Found and fixed this bug, which may matter in grids with large aspect ratios, so I'm thinking I'll also bump to version 0.58.4 after this,

Reply:
A good catch!